src/gps_helper.h

```diff
@@ -151,6 +158,15 @@ class GPSHelper
 	 */
 	virtual int receive(unsigned timeout) = 0;
 
+	/**
+	 * Reset GPS device
+	 * @param restart_type
+	 * @return -1 if not implemented
+	 * 	        0 on failure
```

Comment:
Can you return 0 on success, and <0 for failures?

Reply:
Some GPS drivers do not have reset function, so I changed it like this:
0 : success
-1: not implemented
<0: failure